incubator/jasperserver/templates/jasperserver-pvc.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.persistence.enabled -}}
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: {{ template "fullname" . }}-jasperserver
```

Comment:
Should we add another volume for `/bitnami/tomcat` as it's defined in the docker-compose.yml file of the jasperserver image repo.?

If not, then we should rename this file to simply `pvc.yaml` and change the `name` field to `name: {{ template "fullname" . }}`